README.md

```diff
@@ -701,6 +701,19 @@ $ cd install
 $ stack build
 ```
 
+#### Using Nix
+
+The instructions below show how to set up a Cachix binary cache and open a nix shell for local development.
+
+```shell
+$ cachix use haskell-language-server
+$ nix-shell
+$ cabal update
+$ cabal build
+```
+
+If you are looking for a Nix expression to create haskell-language-server binaries, seehttps://github.com/haskell/haskell-language-server/issues/122
```

Comment:
```suggestion
If you are looking for a Nix expression to create haskell-language-server binaries, see https://github.com/haskell/haskell-language-server/issues/122
```

Reply:
could we include the nix expression in the readme, to being able to close the issue?

Reply:
But it's not a solved problem, and if it is I'm not in a position to state the solution, hence referring to an open issue.